jta-1_2-standalone/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java

```diff
@@ -50,8 +45,6 @@
 
     private MandatoryCounterManager mandatoryCounterManager;
 
-    private TestEntityRepository testEntityRepository;
-
     private Weld weld;
 
     @BeforeClass
```

Comment:
That's how this quickstart was implemented before. All compensations quickstarts are like that too. Why is it bad to show all the APIs via unit tests? Everybody knows how to use them.
That being said, I could add a class with main method which would do the same things as the unit test does.